app/controllers/review_offer/items.js

```diff
@@ -7,6 +7,29 @@ export default Ember.Controller.extend({
   offer: Ember.computed.alias("model"),
   reviewOffer: Ember.inject.controller(),
 
+  lastMessage: Ember.computed(
+    "offer.messages",
+    "offer.messages.[]",
+    function() {
+      return this.get("offer.messages")
+        .sortBy("createdAt")
+        .filter(m => !m.get("isCharityConversation"))
+        .get("lastObject");
+    }
+  ),
+
+  unreadOfferMessagesCount: Ember.computed(
+    "offer.messages",
+    "offer.messages.[]",
+    function() {
+      return this.get("offer.messages")
+        .sortBy("createdAt")
```

Comment:
sortBy can be done after filtering

Reply:
this one can actually be removed altogether, I'll do that